### PR TITLE
Creating event

### PR DESCRIPTION
**Pull request by CreativePhilip**

### CODE REVIEW

File: ippon/models.py [hunk 1]
```diff
@@ -322,3 +323,27 @@ class Tournament(models.Model):
...
     age_constraint_value = models.IntegerField()
     finals_depth = models.IntegerField()
+
+    event = models.ForeignKey("Event", on_delete=models.CASCADE, null=True)
+
+
+class Event(models.Model):
+    name = models.CharField(max_length=200)
+    description = models.TextField()
+    event_owner = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
```
Comment (morynicz) [inline ippon/models.py:334]: This might be an issue. Better add another small table of event owners

File: ippon/test_event_model.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+        registration_start = datetime.datetime.strptime(start_string, '%m/%d/%Y %H:%M:%S')
+        registration_end = datetime.datetime.strptime(end_string, '%m/%d/%Y %H:%M:%S')
+        self.early_event: Event = Event(
+            name="Tournament",
+            description="It's very cool",
+            event_owner=None,
+            registration_start_time=registration_start,
+            registration_end_time=registration_end
+        )
```
Comment (morynicz) [inline ippon/test_event_model.py:34]: Close this in a function instead of copying and pasting multiple times.

File: requirements.txt [hunk 3]
```diff
@@ -31,6 +32,8 @@ mypy==0.770
 mypy-extensions==0.4.3
 packaging==20.3
 permission==0.4.1
+Pillow==7.1.0
+pkg-resources==0.0.0
```
Comment (morynicz) [inline requirements.txt]: Please make sure to not include this package in commits. This seems to be a bug of PyCharm or sth and usually breaks requirements download.

File: ippon_back/settings.py [hunk 4]
```diff
@@ -169,3 +172,6 @@
 EMAIL_HOST_USER = os.environ['MAIL_USER']
 EMAIL_HOST_PASSWORD = os.environ['MAIL_PASSWORD']
 EMAIL_USE_TLS = os.environ['MAIL_USE_TLS'] == 'True'
+
+
+
```
Comment (morynicz) [inline ippon_back/settings.py:177]: Maybe one is enough :)

File: ippon/models.py (hunk 1 above)
Comment (morynicz) [reply ippon/models.py:334]: Also, could you please move it into separate file?

File: requirements.txt (hunk 3 above)
Comment (CreativePhilip) [reply requirements.txt]: From what I've gathered it seems to be a bug in the communication between Ubuntu <-> pip. Fixed this in commit 94dc8d0

File: ippon/test_event_permissions.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+import django.test
+import pytz
+from django.contrib.auth.models import User
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from ippon.models import Event
+
+
+class TestFightPermissions(django.test.TestCase):
```
Comment (morynicz) [inline ippon/test_event_permissions.py]: Copy pasta!

File: ippon/event_models.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    @property
+    def has_started(self) -> bool:
+        now = datetime.datetime.now()
+        if self.start_time.replace(tzinfo=pytz.UTC) < now.replace(tzinfo=pytz.UTC) < self.end_time.replace(tzinfo=pytz.UTC):
+            return True
+        else:
+            return False
+
+
+class EventAdmins(models.Model):
```
Comment (morynicz) [inline ippon/event_models.py:40]: EventAdmiN

**Pull request merged**